cloud_governance/policy/aws/empty_buckets.py

```diff
@@ -9,6 +9,10 @@
 
 
 class EmptyBuckets(NonClusterZombiePolicy):
+    """
+    This class lists and deleted the empty buckets after 7 days of empty,
+    send an alert mail to the user after 4 days of empty.
+    """
 
```

Comment:
This class sends an alert mail for empty bucket to the user after 4 days and delete after 7 days.